.rubocop.yml

```diff
@@ -3,20 +3,23 @@ require:
   - rubocop-performance
 
 AllCops:
-  TargetRubyVersion: 2.3.3
-
-Layout/LineLength:
-  Enabled: false
+  TargetRubyVersion: 2.5
 
 Layout/SpaceBeforeBlockBraces:
   Enabled: false
 
+Layout/LineLength:
+  Max: 120
+
 Metrics/MethodLength:
```

Comment:
do we really don't want this  cop? It is the "dual" of Layout/LineLength ?

Reply:
I prefere keep this rule

Reply:
me too (i don't understand my comment)

Reply:
Just understood my comment... it is not about `Layout/LineLength`! We want to keep `Layout/LineLength` as well. 

However, my comment is about `Metrics/MethodLength`: "don't we want this MethodLength cop?" 

Because at the moment, we define a limit on "horizontal" with LineLength but we dis-activate the limit on "vertical" with MethodLength: false. So we risk having very long methods. 

These two limits work together as duals to limit the complexity and increase the readability, activating only one without the other just distorts the "shape" of the code (i.e., either it become really long vertically or horizontally) and we might want to consider activating MethodLength as well.

Reply:
we are going to keep `Metrics/MethodLength:` with `Max: 30`